nexus/src/internal_api/http_entrypoints.rs

```diff
@@ -277,3 +280,38 @@ async fn cpapi_artifact_download(
 
     Ok(Response::builder().status(StatusCode::OK).body(body.into())?)
 }
+
+/// Endpoint used by Sled Agents to allocate static V6 addresses
+#[endpoint {
+    method = POST,
+    path = "/static_address/v6",
+}]
+async fn allocate_static_v6_address(
+    request_context: Arc<RequestContext<Arc<ServerContext>>>,
+) -> Result<HttpResponseOk<AllocateStaticV6AddressResponse>, HttpError> {
+    let context = request_context.context();
+    let nexus = &context.nexus;
+
+    Ok(HttpResponseOk(AllocateStaticV6AddressResponse {
+        address: nexus.allocate_static_v6_address().await?,
+    }))
+}
+
+/// Endpoint used by Sled Agents to free static V6 addresses
+#[endpoint {
+    method = POST,
+    path = "/static_address/v6/{address}/free",
```

Comment:
We might want this to be a `DELETE` operation, rather than a POST to another endpoint.

Reply:
This is gone now too - the sled agent isn't making these calls anymore.